pangolin.tokenlist.json

```diff
@@ -281,6 +281,14 @@
       "symbol": "CARE",
       "logoURI": "https://raw.githubusercontent.com/pangolindex/tokens/main/assets/19/0xb1D60fA6FABd9A2Ad89960d070350552E47fb0D6/logo_24.png"
     },
+    {
+      "chainId": 19,
+      "address": "0x61b64c643fCCd6ff34Fc58C8ddff4579A89E2723",
+      "decimals": 18,
+      "name": "Astral Credits",
+      "symbol": "XAC",
+      "logoURI": "https://raw.githubusercontent.com/pangolindex/tokens/main/assets/assets/19/0x61b64c643fCCd6ff34Fc58C8ddff4579A89E2723/logo_24.png"
+    }

```

Comment:
there is issue with json here, looks like need comma

Reply:
Updated in new commit.